cluster-up/common.sh

```diff
@@ -22,4 +22,8 @@ set -e
 # the cluster kind is a kubernetes cluster
 if [ ${CLUSTER_PROVIDER} = "kind" ]; then
     CLUSTER_PROVIDER="kubernetes"
+fi
```

Comment:
@marceloamaral , maybe not in this PR, but we always change the CLUSTER_PROVIDER to kubernetes? and this common.sh just do a validation?

Reply:
Right, this is because `make cluster-deploy` access the `kubernetes` folder to deploy de manifests....
And the folder path is created using the `CLUSTER_PROVIDER`....

We could improve that in somehow in the future....